### DISCUSSION TITLE
How to manage multiple bindings

### DISCUSSION THREAD

**Discussion by flodi** [WsdlToPhp/PackageGenerator#320] [Q&A] [answered]
[wsAccesso.txt](https://github.com/user-attachments/files/16443334/wsAccesso.txt)

I have WSDL like the one attached, and I have to use the operation on both binding. How can I specifiy the binding?

**Comment by mikaelcom** [comment, marked as answer]
You don't except that you can change the SoapServer location by calling the setLocation method on the Service instance.
